api/billingd/analytics/analytics.go

```diff
@@ -38,50 +39,51 @@ func NewClient(segmentAPIKey, prefix string, debug bool) (*Client, error) {
 	return client, err
 }
 
-// Update updates the user metadata
-func (c *Client) Update(userID, email string, active bool, properties map[string]interface{}) {
-	if c.api != nil && email != "" {
+// Identify creates or updates the user traits
+func (c *Client) Identify(userID string, accountType mdb.AccountType, active bool, email string, properties map[string]interface{}) error {
```

Comment:
Is _userID_ the customer ID, or something specific to segment?

Reply:
ah, i was just using the segment lingo, but it's just the user pubkey (i.e. `account.Owner().Key`)

Reply:
changed to `key`

Reply:
Cool, sgtm